### ISSUE TITLE
Dummy Posts for Presentation

### ISSUE THREAD

**Issue by AlexZadorojnyi** [peterpasz/ShareScape#15]
I was thinking pics of stuff around the Ryerson area. These don't have to be posted in the actual locations since we can just manually enter posts in the database.

**Comment by peterpasz** [comment]
Yea i was thinking the same thing. We can take pictures of some landmarks or restaurants. The ryerson statue guy, pic of slc, pic of dundas square, etc.

**Comment by peterpasz** [comment]
Also, nhan was mentioning that we can have videos demoing parts of our app. So we could show it off live, but have videos prepated as well. I think I've got an app on my phone that records screen so i could show off the mobile version. If anyones got frapps or something for recording footage of their laptop's screen, we could do it for that as well.

**Comment by AlexZadorojnyi** [comment]
Yeah, I agree, having a pre-recorded bit for at least the mobile version is a good idea.
